pipelines/dotnet.yml

```diff
@@ -27,12 +28,105 @@ steps:
     command: restore
     projects: '$(package)'
 
+- powershell: |
+    # This allows us to not have to checkin .csproj files with DelaySign and SignAssembly set to to true. If the flag is set,
+    # then project is not debuggable with SignAssembly set to true.
+    $csprojPath = @(".\dotnet\packages\Microsoft.TeamsAI\Microsoft.TeamsAI\Microsoft.Teams.AI.csproj")
+    $doc = New-Object System.Xml.XmlDocument
+    $doc.Load($csprojPath)
+    # Set the DelaySign element to 'true' so that delay signing is set.
+    $delaySign = $doc.SelectSingleNode("//DelaySign");
```

Comment:
You should use PublicSign instead of DelaySign. 

Read about PublicSign here:
https://github.com/dotnet/runtime/blob/main/docs/project/public-signing.md